File: javascript/Shubham Sigdar/Basic Mathematical Operations.js
//Your task is to create a function that does four basic mathematical operations.

// The function should take three arguments - operation(string/char), value1(number), value2(number).
// The function should return result of numbers after applying the chosen operation.

// Examples(Operator, value1, value2) --> output
// ('+', 4, 7) --> 11
// ('-', 15, 18) --> -3
// ('*', 5, 5) --> 25
// ('/', 49, 7) --> 7
//P:string? number? special character?
//R:return result of numbers after applying the chosen operation.
//E:
//P:
function basicOp(operation, value1, value2)
{
  switch(operation){
    case "+":
      return value1 + value2
      break
    case "-":
      return value1 - value2
      break
    case "*":
      return value1 * value2
      break
    case "/":
      return value1 / value2
      break  
  }
}
console.log(basicOp('+', 4, 7), 11)
console.log(basicOp('-', 15, 18), -3)
console.log(basicOp('*', 5, 5), 25)
console.log(basicOp('/', 49, 7), 7)

File: java/Shubham Sigdar/BasicMathematicalOperations.java
//Your task is to create a function that does four basic mathematical operations.
//
//        The function should take three arguments - operation(string/char), value1(number), value2(number).
//        The function should return result of numbers after applying the chosen operation.
//
//        Examples(Operator, value1, value2) --> output
//        ('+', 4, 7) --> 11
//        ('-', 15, 18) --> -3
//        ('*', 5, 5) --> 25
//        ('/', 49, 7) --> 7
public class BasicMathematicalOperations {
    public static Integer basicMath(String op, int v1, int v2)
    {

        if(op.equals("+")){
            return v1 + v2;
        }else if(op.equals("*")){
            return v1 * v2;
        }else if(op.equals("/")){
            return v1 / v2;
        }else{
            return v1 - v2;
        }
    }
}
